src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/PrisonDto.kt

```diff
@@ -17,6 +18,19 @@ data class PrisonDto(
   @Schema(description = "maximum number of days notice from the current date to booked a visit", example = "28", required = true)
   var policyNoticeDaysMax: Int,
 
+  @Schema(description = "Max number of total visitors")
+  @field:Min(1)
+  val maxTotalVisitors: Int,
+  @Schema(description = "Max number of adults")
+  @field:Min(1)
+  val maxAdultVisitors: Int,
+  @Schema(description = "Max number of children, if -1 then no limit is applied")
+  @field:Min(-1)
+  val maxChildVisitors: Int,
+  @Schema(description = "Age of adults in years")
+  @field:Min(10)
```

Comment:
I think this needs to be 17  will bring up the minimum age for adults tomorrow in standup